lib/carrierwave/storage.rb

```diff
@@ -2,8 +2,9 @@
 require "carrierwave/storage/file"
 
 begin
-  require "fog" unless defined?(::Fog)
+  require "fog/core"
 rescue LoadError
+  require "fog" unless defined?(::Fog)
```

Comment:
Doesn't this require fog as a runtime dependency? Need to still keep it optional.